#EXTRA FUNCTION

Header header

uint8 obs_avoid_enable
uint8 laser_height_enable
uint8 add_one
uint8 add_two
uint8 add_three
